packages/bus/project.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Create the publishable package as stayed in #1. You just need to issue the following command right after `ng generate`:
```
$ nx workspace-generator publish package-name --buildTarget package-name:build
```